twelvelabs/models/video.py

```diff
@@ -39,11 +39,18 @@ class VideoValue(BaseModel):
     value: str
 
 
+class VideoSource(BaseModel):
+    type: str
+    name: str
+    url: Optional[str] = None
+
+
 class Video(ObjectWithTimestamp):
     _resource: VideoResource = PrivateAttr()
     _index_id: str = PrivateAttr()
     metadata: VideoMetadata
```

Comment:
what is the reason that timestamp fields like created_at, indexed_at are not included here?

Reply:
just added timestamp fields in video object, I missed!